### ISSUE TITLE
Panic on DTLS traffic prior to negotiation

### ISSUE THREAD

**Issue by OxleyS** [algesten/str0m#494]
Inspired by the recent discussions on fuzzing and not panicking on direct network input, I did a silly thing and hacked my code such that all clients' traffic was passed to all `Rtc` instances without calling `Rtc.accepts()`, just to see what would happen 😄 

I ended up getting a panic instantly:
```
Message:  set_active must be called
Location: [...]/str0m-3b15da0a44f6bcc8/a0f4217/src/crypto/ossl/stream.rs:79
```

This seems to happen when client B gets client A's DTLS traffic, but client B hasn't gotten SDP from the remote end yet (which is what calls `set_active`).

I expect that an `Rtc.accepts()` guard would reject this traffic because with no remote SDP, you can't have any candidate pairs either. However, [the documentation](https://github.com/algesten/str0m/blob/a0f42179c6936962672413ee38da63968eb91e06/src/lib.rs#L1471-L1472) implies that if you aren't UDP socket multiplexing, you don't need to call this.

Is this documentation correct? If so, what should we do about the panic? If we don't want to eat the overhead of checking candidate pairs on every `handle_input` call, we could simply insert an early-out on DTLS traffic prior to `set_active` being called.

**Comment by algesten** [comment]
Yeah. I think you're right. In `OsslDtlsImpl::handle_receive` we can add a check for `self.is_active()`. It's an `Option<bool>` and we don't consider the DTLS subsystem fully turned on until that is `Some`, which is what `is_active()` checks for.

PR welcome!
